packages/postcss-minify-params/src/index.js

```diff
@@ -34,7 +34,14 @@ function transform (legacy, rule) {
     // at-rule. For example:
     //
     // @value vertical, center from "./foo.css";
-    if (!rule.params || rule.name.toLowerCase() === 'value') {
+    //
+    // We should also not re-arrange pseudo-classes for @page at-rule.
+    // For example:
+    //
+    // @page :first { margin: 0; }
+    const ruleName = rule.name.toLocaleLowerCase();
+    if (!rule.params ||
+        rule.type === 'atrule' && (ruleName === 'value' || ruleName === 'page')) {
```

Comment:
Good job! no need `rule.type === 'atrule'`, because we already in `walkAtRules` and all rules here are `atrule`

Reply:
@evilebottnawi Thank you for pointing this out, I fixed it and pushed the changes.